innative/parse.cpp

```diff
@@ -150,12 +151,13 @@ IN_ERROR innative::ParseFunctionType(Stream& s, FunctionType& sig, const Environ
 
 IN_ERROR innative::ParseResizableLimits(Stream& s, ResizableLimits& limits)
 {
+  // This will only actually read a single byte
```

Comment:
Why is this using ParseVarUInt32 if it will only actually read a single byte? If there is only one byte it should be varuint7, unless the spec has done something very strange.

Reply:
I'm actually not sure on this one tbh. Technically according to the spec the only valid values here are {0,1,3}, but it was already reading a varuint32. I can change it to varint7 if you'd prefer though